spec/acceptance/php_spec.rb

```diff
@@ -45,7 +53,10 @@
               package_prefix => 'php-',
               settings       => {
                 extension => undef
-              }
+              },
+            }
+            'simplexml'  => {
+              package_name => simplexmlpackagename,
```

Comment:
the value of a hash key needs to be quoted. In this case:
```suggestion
              package_name => 'simplexmlpackagename',
```

But I don't think that `simplexmlpackagename` is a valid name for any package on any distribution?

Reply:
Hi,
  package name for simplexml depends on used OS (php7.2-xml for ubuntu 1804, php7.0-xml for ubuntu 1604 and php-xml for ubutu 1404).

I am trying set the correct name by switch (see https://github.com/voxpupuli/puppet-php/pull/503/files#diff-2e2ff95fcba379972e19088c70dd68a5R39 ), but it not work. Can you help me with this, please?

Reply:
ahh my bad. This should work:
```suggestion
              package_name => '#{simplexmlpackagename}',
```